stmt/stmts/insert.go

```diff
@@ -46,15 +46,20 @@ const (
 	DelayedPriority
 )
 
+// InsertRest is the rest part of insert/replace into statement.
+type InsertRest struct {
```

Comment:
I prefer remove InsertRest to rsets package, maybe this or nexr PR.
@siddontang  How about u?